core/src/main/java/org/mobilitydata/gtfsvalidator/input/GtfsUnarchivedInput.java

```diff
@@ -0,0 +1,36 @@
+package org.mobilitydata.gtfsvalidator.input;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Implements support for unarchived GTFS directories.
+ */
+public class GtfsUnarchivedInput implements GtfsInput {
+    private final Set<String> filenames = new HashSet();
+    private final File directory;
+
+    public GtfsUnarchivedInput(File directory) throws IOException {
+        this.directory = directory;
+        for (File file : directory.listFiles()) {
+            if (!file.isDirectory()) {
+                filenames.add(file.getName());
+            }
+        }
+
```

Comment:
Nit - extra new line

Reply:
Removed. I wonder why "Reformat code" in IDEA does not drop such redundant lines.